src/cluster/serializer/broker/Serializer.cc

```diff
@@ -85,7 +85,7 @@ std::optional<broker::zeek::Event> detail::to_broker_event(const detail::Event&
                 broker::vector entry(2);
                 entry[0] = static_cast<broker::count>(m.Id());
                 entry[1] = res.value();
-                broker_meta.push_back(std::move(entry));
+                broker_meta.emplace_back(std::move(entry));
```

Comment:
Oh oh, I might have missed a cleanup suggestion from Benjamin - mind changing this to:

```
                broker::vector entry{{static_cast<broker::count>(m.Id()), res.value()}};
                broker_meta.emplace_back(std::move(entry));
```

Reply:
Or even better `broker::vector entry{static_cast<broker::count>(m.Id()), res.value()};` unless we wanted to mix uniform initialization and explicitly creating an initializer list where `std::vector` can already directly construct from an initializer list ;)